src/data/complex/basic.lean

```diff
@@ -111,6 +112,8 @@ def I : ℂ := ⟨0, 1⟩
 @[simp] lemma I_im : I.im = 1 := rfl
 
 @[simp] lemma I_mul_I : I * I = -1 := ext_iff.2 $ by simp
+lemma I_mul (a b : ℝ) : I * ⟨b, -a⟩ = ⟨a, b⟩ :=
```

Comment:
This isn't the right name for this lemma. If the statement were `I * ⟨a, b⟩ = ⟨-b, a⟩` you could call it `I_mul_mk`.

Reply:
Alternatively, it keeps the name, but the statement becomes
```
lemma I_mul (z : complex) : I * z = ⟨-z.im, z.re⟩ :=
```
Note that in lean, `z` is (unfortunately) not defeq to `⟨z.re, z.im⟩`, so you might have trouble applying your statement in situations that are not exactly of the form you wrote it in.